Source/SpacePark/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using SpacePark.Library.Context;
+using SpacePark.Library.Models;
+
+namespace SpacePark
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            using var context = new SpaceParkContext();
+
+            // Creates a brand new SpacePort Garage if we do not have one already
+            SpacePort spacePort = CreateSpacePort(context);
+
+            CheckParkingSpaces(context, spacePort);
+
+            CreateHeader();
+
+            await DisplayMenu(context);
+        }
+
+        private static async Task DisplayMenu(SpaceParkContext context)
+        {
+            while (true)
+            {
+                Console.WriteLine("Press 1 to park: ");
+                Console.WriteLine("Press 2 to pay: ");
+                Console.WriteLine();
+                Console.Write(">>> ");
+                var userChoice = Convert.ToInt32(Console.ReadLine());
+
+                if (userChoice == 1)
+                {
+                    Console.Clear();
+                    await RentParkingSpace(context);
+                }
+                else
+                {
+                    Console.Clear();
+                    await ClearParkingSpace(context);
+                }
+            }
+        }
+
+        private static async Task ClearParkingSpace(SpaceParkContext context)
+        {
+            // Not yet implemented!
+            Console.Write("Name: ");
+            var visitorName = Console.ReadLine();
+            Console.WriteLine(visitorName);
```

Comment:
Kan använda en exception för not yet implemented här istället för att ha "placeholder" kod.